zcash_client_backend/src/constants/regtest.rs

```diff
@@ -0,0 +1,45 @@
+//! # Regtest constants
+//!
+//! `regtest` is a `zcashd`-specific environment used for local testing. They mostly reuse
+//! the testnet constants.
+//! These constants are defined in [the `zcashd` codebase].
+//! [the `zcashd` codebase]: https://github.com/zcash/zcash/blob/128d863fb8be39ee294fda397c1ce3ba3b889cb2/src/chainparams.cpp#L482-L496
+
+/// The regtest cointype reuses the testnet cointype
+pub const COIN_TYPE: u32 = 1;
```

Comment:
Given that this reuses the testnet cointype, this might be better as:
```suggestion
pub const COIN_TYPE: u32 = super::testnet::COIN_TYPE;
```